packages/web/components/denom/types.ts

```diff
@@ -0,0 +1,7 @@
+import { AppCurrency, Currency } from "@keplr-wallet/types";
+
+export interface DenomImageProps {
```

Comment:
Opinion: I like to inline props in the component unless they are particularly reusable. Like with `ModalBaseProps`. Generally, that's how I like to approach all type definitions.

Reply:
Typescript is great, but I want to maintain and grok code, not types

Reply:
Thanks for the review! 

> Opinion: I like to inline props in the component unless they are particularly reusable. Like with `ModalBaseProps`. Generally, that's how I like to approach all type definitions.

Good to know. I thought `types.ts` files were the convention for all types in general including components. Will update this.